plugins/woocommerce/changelog/feature-marketplace-subscriptions

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: update
+
+The WooCommerce > Extensions section now includes a rebuilt "My Subscriptions" section with improvements to how subscriptions can be managed and installed.
```

Comment:
I'm in two minds whether this is a "minor" or a "major", but it certainly feels "major" based on the amount of changes involved.

Just to avoid the repetition of "section".

```suggestion
The WooCommerce > Extensions section now includes a rebuilt "My Subscriptions" page with improvements to how subscriptions can be managed and installed.
```

Reply:
Yeah, I agree. The scope is bordering major so I went with that and figured we could be corrected if it comes up.